src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/EntityReferenceItem.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\Field\FieldTypeExtend;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\Plugin\Field\FieldType\EntityReferenceItem as EntityReferenceItemExtend;
+
+/**
+ * {@inheritdoc}
+ */
+class EntityReferenceItem extends EntityReferenceItemExtend {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    $values = parent::generateSampleValue($field_definition);
+    switch ($field_definition->getName()) {
+      // Random values must be unique.
+      case 'field_subjects':
+        static $unique = [];
+        // 25 subject terms are generated and the maximum number associated with content is 3.
```

Comment:
not sure what the comment says, the behavior of the class seems to be that we get to up to 10 items in `$unique`, then we only keep the last 2 and go on. The return value seems to be only 1 target.